crates/reliable-broadcast/src/lib.rs

```diff
@@ -44,11 +55,21 @@ where
     ) -> impl Future<Output = S::Aggregated> {
         let receivers: Vec<_> = self.validators.clone();
         let network_sender = self.network_sender.clone();
+        let time_service = self.time_service.clone();
+        let mut backoff_policies: HashMap<Author, TBackoff> = self.validators
```

Comment:
So this allows different backoff policies for different nodes, but here it uses the same policy for all nodes? Should we support different policies, or update the policy?

Reply:
We will have the same policy for different peers. TBackoff is both a policy definition and an implementation - it gives the next interval to wait for each peer, so we need a copy for each one of them.

Reply:
Why do you have a map if all have the same policy?

Reply:
because each one maintains the next interval to wait for. we need one for each peer. 

Otherwise, if we have two peers and one exponential backoff, then both of them have to retried, then for first peer we will use interval*factor but for second peer interval*factor*factor which is not what we want.

Reply:
Right, I am stupid, ignore me.